include/xtensor/xdynamic_view.hpp

```diff
@@ -693,7 +693,7 @@ namespace xt
         using slice_vector = typename view_type::slice_vector_type;
         using policy = detail::adj_strides_policy<slice_vector>;
         detail::strided_view_args<policy> args;
-        args.fill_args(e.shape(), detail::get_strides(e), detail::get_offset(e), e.layout(), slices);
+        args.fill_args(e.shape(), detail::get_strides<XTENSOR_DEFAULT_TRAVERSAL>(e), detail::get_offset<XTENSOR_DEFAULT_TRAVERSAL>(e), e.layout(), slices);
```

Comment:
Use layout of flat_storage_getter?

Reply:
TODO: open issue to use `flat_storage_getter` instead of `flat_expression_adaptor`?